internal/rpcserver/rpcwebsocket.go

```diff
@@ -1524,9 +1524,10 @@ out:
 				// Check credentials.
 				login := authCmd.Username + ":" + authCmd.Passphrase
 				auth := "Basic " + base64.StdEncoding.EncodeToString([]byte(login))
-				authSha := sha256.Sum256([]byte(auth))
-				cmp := subtle.ConstantTimeCompare(authSha[:], c.rpcServer.authsha[:])
-				limitcmp := subtle.ConstantTimeCompare(authSha[:], c.rpcServer.limitauthsha[:])
+				mac := make([]byte, 0, sha256.Size)
+				mac = c.rpcServer.authMAC(mac, []byte(auth))
+				cmp := subtle.ConstantTimeCompare(mac[:], c.rpcServer.authsha[:])
+				limitcmp := subtle.ConstantTimeCompare(mac[:], c.rpcServer.limitauthsha[:])
```

Comment:
mac is already []byte and doesn't need to be sliced again (I thought this was supposed to be caught by gosimple?)

Reply:
Good catch.  Updated.